src/views/clusteroverview/overview/components/top-consumers-card/utils/DurationDropdown.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+
+import { Select, SelectOption, SelectVariant } from '@patternfly/react-core';
+
+import DurationOption from './DurationOption';
+
+export type DurationDropdownProps = {
+  selectedDuration: any;
+  selectHandler: any;
```

Comment:
can you please change the type from `any`?

Reply:
Done. Another good catch. I need to start adding TODOs when I mean to go back and fix something. My memory isn't good enough to just remember them, anymore. =)